lib/explorer/series.ex

```diff
@@ -6020,6 +6020,29 @@ defmodule Explorer.Series do
     apply_series(series, :transform, [fun])
   end
 
+  @doc """
+  Extracts a field from a struct series
+
+  ## Examples
+
+      iex> s = Series.from_list([%{a: 1}, %{a: 2}])
+      iex> Series.field(s, "a")
+      #Explorer.Series<
+        Polars[2]
+        s64 [1, 2]
+      >
+  """
+  @doc type: :struct_wise
+  @spec field(Series.t(), Explorer.Backend.Series.valid_types()) :: Series.t()
+  def field(%Series{dtype: {:struct, dtype}} = series, name) do
```

Comment:
Aren't struct fields all strings?

```suggestion
  @spec field(Series.t(), String.t()) :: Series.t()
  def field(%Series{dtype: {:struct, dtype}} = series, name) when is_binary(name) do
```